js/ui/list.js

```diff
@@ -1,10 +1,12 @@
-import ListEdit from './list/ui.list.edit.search';
-import registerComponent from '../core/component_registrator';
-// STYLE list
-
-registerComponent('dxList', ListEdit);
+import List from './list_light';
+import './list/modules/selection';
+import './list/modules/deleting';
+import './list/modules/dragging';
+import './list/modules/context';
+import './list/modules/search';
```

Comment:
I don't really like an idea of implicit plugin usage since this may be not intuitive and the code may be accidentally removed during refactoring
```suggestion
import selection from './list/modules/selection';
import deleting from './list/modules/deleting';
import dragging from './list/modules/dragging';
import context from './list/modules/context';
import search from './list/modules/search';

const ListEdit = List
    .extend(selection)
    .extend(deleting)
    .extend(dragging)
    .extend(context)
    .extend(search)
    
export default ListEdit;
```

Reply:
I still think that simple is better. Will discuss it separately.